01_prepare_host.sh

```diff
@@ -256,7 +256,7 @@ function configure_minikube() {
 #
 function init_minikube() {
```

Comment:
Remove all `function `, it is not needed for anything. `something() {` is correct.